cp-styles/app/styles/components/_no-results.scss

```diff
@@ -44,3 +44,5 @@ table .field-no-results {
 	text-align: center;
 	padding: $sq-inset-s;
 }
+
+.field-no-results.hidden { display: none !important;}
```

Comment:
can you rewrite here using SASS syntax? (i.e. nesting the selectors)

Reply:
rewrote code